tests/security/cc/cc_disable_root_ssh.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Disable root ssh login due to CC hard requirement
```

Comment:
Disable or Enable?

Reply:
Good question, Disable -> CC setup, root ssh login is not permitted.